bazel-bin

```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_martinbomio/17e5cf616981f27d03c506e3e9f0879d/execroot/tfx_bsl/bazel-out/darwin-opt/bin
```

Comment:
could you please revert files outside of the project

Reply:
oops, removed them in https://github.com/tensorflow/tfx-bsl/pull/54/commits/a65d12b8d76ff70d578475867ebccd03f59e02e7

Reply:
imported internally, under review